From e440889dfd25c5d4328cf4624f0379e065f895fd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Uwe=20Kleine-K=C3=B6nig=20=28The=20Capable=20Hub=29?=
 <u.kleine-koenig@baylibre.com>
Date: Fri, 3 Jul 2026 17:46:18 +0200
Subject: [PATCH 13274/15333] nfc: Unify style of acpi_device_id arrays
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Unify the style of the list terminator in acpi_device_id arrays, that is
use a single space between { and }. This is the most common and
generally recommended style for these.

Signed-off-by: Uwe Kleine-König (The Capable Hub) <u.kleine-koenig@baylibre.com>
Link: https://patch.msgid.link/30cd94a6821917f16daffdce5fabf145432c13eb.1783091699.git.u.kleine-koenig@baylibre.com
Signed-off-by: David Heidelberg <david@ixit.cz>
---
 drivers/nfc/fdp/i2c.c      | 2 +-
 drivers/nfc/pn544/i2c.c    | 2 +-
 drivers/nfc/st-nci/i2c.c   | 2 +-
 drivers/nfc/st-nci/spi.c   | 2 +-
 drivers/nfc/st21nfca/i2c.c | 2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/nfc/fdp/i2c.c b/drivers/nfc/fdp/i2c.c
index 47f3838ac9d4..13d4387e79a0 100644
--- a/drivers/nfc/fdp/i2c.c
+++ b/drivers/nfc/fdp/i2c.c
@@ -350,7 +350,7 @@ static void fdp_nci_i2c_remove(struct i2c_client *client)
 
 static const struct acpi_device_id fdp_nci_i2c_acpi_match[] = {
 	{ .id = "INT339A" },
-	{}
+	{ }
 };
 MODULE_DEVICE_TABLE(acpi, fdp_nci_i2c_acpi_match);
 
diff --git a/drivers/nfc/pn544/i2c.c b/drivers/nfc/pn544/i2c.c
index ccca07eb828c..9ed1cde1de2e 100644
--- a/drivers/nfc/pn544/i2c.c
+++ b/drivers/nfc/pn544/i2c.c
@@ -52,7 +52,7 @@ MODULE_DEVICE_TABLE(i2c, pn544_hci_i2c_id_table);
 
 static const struct acpi_device_id pn544_hci_i2c_acpi_match[] = {
 	{ .id = "NXP5440" },
-	{}
+	{ }
 };
 MODULE_DEVICE_TABLE(acpi, pn544_hci_i2c_acpi_match);
 
diff --git a/drivers/nfc/st-nci/i2c.c b/drivers/nfc/st-nci/i2c.c
index 3906a806ec8b..f43ae8e92070 100644
--- a/drivers/nfc/st-nci/i2c.c
+++ b/drivers/nfc/st-nci/i2c.c
@@ -265,7 +265,7 @@ MODULE_DEVICE_TABLE(i2c, st_nci_i2c_id_table);
 static const struct acpi_device_id st_nci_i2c_acpi_match[] = {
 	{ .id = "SMO2101" },
 	{ .id = "SMO2102" },
-	{}
+	{ }
 };
 MODULE_DEVICE_TABLE(acpi, st_nci_i2c_acpi_match);
 
diff --git a/drivers/nfc/st-nci/spi.c b/drivers/nfc/st-nci/spi.c
index 8fc60a972d98..9303217acd7b 100644
--- a/drivers/nfc/st-nci/spi.c
+++ b/drivers/nfc/st-nci/spi.c
@@ -279,7 +279,7 @@ MODULE_DEVICE_TABLE(spi, st_nci_spi_id_table);
 
 static const struct acpi_device_id st_nci_spi_acpi_match[] = {
 	{ .id = "SMO2101" },
-	{}
+	{ }
 };
 MODULE_DEVICE_TABLE(acpi, st_nci_spi_acpi_match);
 
diff --git a/drivers/nfc/st21nfca/i2c.c b/drivers/nfc/st21nfca/i2c.c
index e5109f701795..13fb6f5533e0 100644
--- a/drivers/nfc/st21nfca/i2c.c
+++ b/drivers/nfc/st21nfca/i2c.c
@@ -579,7 +579,7 @@ MODULE_DEVICE_TABLE(i2c, st21nfca_hci_i2c_id_table);
 
 static const struct acpi_device_id st21nfca_hci_i2c_acpi_match[] = {
 	{ .id = "SMO2100" },
-	{}
+	{ }
 };
 MODULE_DEVICE_TABLE(acpi, st21nfca_hci_i2c_acpi_match);
 
-- 
2.55.0

